roles/config-vlans/README.md

```diff
@@ -0,0 +1,93 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with prepping for a vLAN (virtual LAN) install and configuring a vLAN install. VLANs allow for creating virtual area networks to avoid the costly process of buying and configuring new expensive routers for what may be a temporary measure.
+
+The role is relatively straightforward:  it configures the vLANS interfaces and brings them online.
+
+Requirements
+------------
+
+None beyond ansible.
+
+Role Variables
+--------------
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**ifcfg.device**| the home directory where all the VNC files live | yes | ``` /home/ ``` |
```

Comment:
Please update this table with all the parameters that the role accepts (remember to check the template used to create the ifcfg file for the VLAN).